colossus-examples/src/main/scala/colossus-examples/WebsocketExample.scala

```diff
@@ -82,13 +83,17 @@ object WebsocketExample {
           case "MANY" => {
```

Comment:
Yes, every connection gets a unique proxy actor.  Also in this case "MANY" will block only until the output buffer fills up (which I believe defaults to 50).  When it comes to blocking, it's true that while the output buffer is being filled, nothing else can occur.  In this situation it doesn't matter since it happens very quickly.  I'd imagine if generating the messages to send required significant CPU usage, it would be done on a different thread.

I have thought about ways to have more control over the "greediness" of pipes and situations like this.  One potential is to create a "deferred" wrapper for a pipe which will force the return of a "Full" result after a certain number of items have been pulled and fire off the returned signal at the beginning of the next event loop iteration (using a CallbackExecutor).  This would let you, for example, force the "MANY" situation to only send one item per event loop iteration.

I think in most cases when it comes to dealing with a "Full" vs "Ok" result, you just want to wait until the underlying buffer can accept more items and continue from there.  In fact, the code here can be cleaned up a lot by just wrapping this in an iterator and forwarding it into the output buffer directly using `Source.into`.  I may update or add to this example to show that.